README.md

```diff
@@ -66,6 +66,7 @@ If you have questions or comments, please create an issue.
 - ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - This comprehensive article is directed towards businesses (but still useful for individual contributors) where it talks about why, how, and what open-source projects to contribute to.
 - ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -
 Lets Git started in the world of opensource, starting in the opensource playground. Especially designed for education and practical experience purposes.
+- ["getting-started-with-open-soruce" by NumFocus](https://github.com/numfocus/getting-started-with-open-source) - a Github repo that helps contributors overcoming barriers in open-source.
```

Comment:
Great work! Just a slight suggestion to make the format consistent with the rest of the section titles and there was a slight typo too.

```suggestion
- ["Getting Started with Open Source" by NumFocus](https://github.com/numfocus/getting-started-with-open-source) - a GitHub repo that helps contributors overcome barriers to entry in open-source.
```